R/tcplObj.R

```diff
@@ -62,7 +62,7 @@ cnst = function(ps,x){
   return(rep(0,length(x)))
 }
 
-#' Exponential 2 Model - ps[1]*(exp(x/ps[2]) - 1)
+#' Exponential 2 Model - \eqn{f(x) = a*(e^{(x/b)}- 1)}
```

Comment:
Suggest just putting the equation on a separate line from the title/model name.  Apply this to the other functions in this file.

Reply:
Changes made. The latest rendered pdf had been pushed